### DISCUSSION TITLE
Keisei Integration

### DISCUSSION THREAD

**Discussion by shdwns** [Sepitus-exe/WKElementaryDark#133] [Ideas]
It would be great if this theme supported Keisei Userscript https://community.wanikani.com/t/userscript-keisei-形声-semantic-phonetic-composition/21479 since it is such a useful script!

**Comment by wrex** [comment]
Please note that the script itself needs to be updated. Please see this comment on the forums: https://community.wanikani.com/t/userscript-keisei-%E5%BD%A2%E5%A3%B0-semantic-phonetic-composition/21479/386?u=rrwrex
